server/services/store/storetests/users.go

```diff
@@ -52,3 +68,99 @@ func testGetWorkspaceUsers(t *testing.T, store store.Store) {
 		require.Nil(t, err)
 	})
 }
+
+func testCreateAndGetUser(t *testing.T, store store.Store) {
+	user := &model.User{
+		ID:       uuid.New().String(),
+		Username: "damao",
+		Email:    "mock@email.com",
+	}
+
+	t.Run("CreateUser", func(t *testing.T) {
+		err := store.CreateUser(user)
+		require.Nil(t, err)
```

Comment:
```suggestion
		require.NoError(t, err)
```
Same for the other instances below.